modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,86 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} CLI:
+
+. Configure the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
+----
++
+The namespace must match the `targetNamespaces` value from the OperatorGroup.
+
+. Configure the name of the OpenShift Update Service application, for example, `service`:
++
+[source,terminal]
+----
+$ NAME=service
+----
+
+. Configure the local registry and repository for the release images, for example, `registry.example.com/openshift/release`:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/openshift/release
+----
+
+. Set the local pullspec for the graph-data image to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`:
++
+.Example output
+[source,terminal]
+----
+NAME                             AGE
+openshift-update-service-fprx2   4m41s
```

Comment:
The example output seems like the output of next step.

Reply:
fixed